src/Ownership/Readmodels/OwnershipPermissionProjector.php

```diff
@@ -147,7 +151,7 @@ private function createRole(OwnershipItem $ownershipItem): UUID
         $this->commandBus->dispatch(
             new CreateRole(
                 $roleId,
-                $this->createRoleName($ownershipItem->getId())
+                'Beheerders ' . $this->itemNameResolver->resolve($ownershipItem->getItemId())
```

Comment:
Is this a public visible name?
Doesn't UDB have multiple languages? This will never be translated?
Shouldn't we add this label to the frontend, instead of hardcoding it in one language?

Reply:
The role name is always in one language, there are no translations for role names.

Reply:
That being said, the role name itself, is never exposed to a normal user. This role, in fact, all roles are only visible in an admin view for users who have this very limited permission.